app/views/users/show.html.erb

```diff
@@ -0,0 +1,33 @@
+<h3> Your Events </h3>
+<p>
+   <ul id = "users">
+      <% @user.created_events.each do |event|%>
+      <li>
+          Event: <%= event.name %> <br/> 
+      </li>
+      <li>
+        Location: <%= event.location %><br/>
+      </li>
+      <li>     
+         Event_Date: <%= event.start_date %>
+      </li>
+      <%end%>    
+   </ul> 
+</p>
+<h3>Future Events</h3>
+<ul>
+   <%@user.attended_events.each do |event| %>
```

Comment:
- [ ] As was pointed out before, the logic for future events should be in your user controller. 
Screenshot from Odin project 👇 
<img width="834" alt="Screen Shot 2020-12-06 at 10 18 58 AM" src="https://user-images.githubusercontent.com/48804258/101284129-757d9880-37ac-11eb-9580-c2d27f98641e.png">

- **Note where it says you could put it in your views but you should not do that 👇** 
<img width="782" alt="Screen Shot 2020-12-06 at 10 20 08 AM" src="https://user-images.githubusercontent.com/48804258/101284228-ef158680-37ac-11eb-928b-bb07198853ba.png">